### PR TITLE
TRTLLM sample notebook

### PR DESCRIPTION
**Pull request by ydm-amazon**

### CODE REVIEW

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a174b36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_uri = image_uris.retrieve(\n",
+    "        framework=\"djl-deepspeed\",\n",
```
Comment (lanking520) [inline aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb:116]: this is not right

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+   "id": "b011bf5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile serving.properties\n",
+    "engine=MPI\n",
+    "option.model_id=TheBloke/Llama-2-13B-fp16\n",
+    "option.tensor_parallel_degree=4\n",
+    "option.max_rolling_batch_size=64\n",
+    "option.entryPoint=djl_python.tensorrt_llm\n",
```
Comment (lanking520) [inline aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb:70]: we don't need to set entrypoint

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    "option.rolling_batch=trtllm\n",
+    "option.dtype=fp16"
```
Comment (lanking520) [inline aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb:72]: dtype is defaulted to fp16 from V7

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb (hunk 1 above)
Comment (ydm-amazon) [reply aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb:116]: Will fix

Comment (ydm-amazon) [reply aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb]: Done

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb (hunk 2 above)
Comment (ydm-amazon) [reply aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb]: Done

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb (hunk 3 above)
Comment (ydm-amazon) [reply aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb]: Done

File: aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a174b36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image_uri = \"763104351884.dkr.ecr.us-east-1.amazonaws.com/djl-inference:0.25.0-tensorrtllm0.5.0-cu122\""
```
Comment (lanking520) [inline aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb:113]: you need to fetch from SageMaker pySDK. Use our staging build for your test

Comment (ydm-amazon) [reply aws/sagemaker/large-model-inference/sample-llm/trtllm_rollingbatch_deploy_llama_13b.ipynb]: Done.

**Comment by lanking520** [comment]
could you add a link to your notebook in the outer folder README.md?

**Review by lanking520 — APPROVED** [review]

**Pull request merged**